packages/iris-grid/src/ColumnStatistics.scss

```diff
@@ -23,6 +23,10 @@
   white-space: nowrap;
 }
 
+.column-statistics-status {
+  margin: $spacer-1 0;
```

Comment:
In the screenshot on the ticket, the left side of the lock is aligned with the text. Adjust the margin to align it correctly.
The "Generate Statistics" text should also left align, may as well fix that up while you're in there as well.